src/contributing.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: layouts/base.njk
+---
+
+Welcome! First off; _**THANK YOU**_ for your interest in contributing to Zinc!
+Your time is valuable, so let's get started!
+
+## Ground Rules
+
+Zinc moves forward thanks to two kinds of workers. _Contributors_, who suggest
+and build changes to our software; and _Maintainers_, who decide if, when, and
+how to apply the suggestions. Anyone and everyone can be a Contributor! All we
+ask is that Contributors follow a few [Ground Rules][contributor-ground-rules]
+to help Zinc Maintainers and Contributors work well together.
+
+The keywords _must_, _must not_, and _should_ in this document are to be
+interpreted as described in [RFC 2119][rfc-2119].
+
+### Contributor Ground Rules
+
+- Contributors _must_ comply with Zinc's [code of conduct][code-of-conduct].
+- Contributors _must_ [assign ownership][assign-ownership] of their
+  contributions to Zinc.
+- Contributors _should_ [ask for help][get-help] when they get stuck.
+- Contributors _should_ expect others to build their Contributions.
+
+### Maintainer Ground Rules
+
+- Maintainers _must_ comply with Zinc's [code of conduct][code-of-conduct].
+- Maintainers _must_ ensure that all feedback provided to Contributors is
+  specific, actionable, and kind.
```

Comment:
Must + all is too strong here IMO.

If I give some specific comments and then an overall "good job!", that bit of feedback isn't specific or actionable. Also, "kind" is somewhat subjective, and striving to make sure every bit of feedback is kind sounds difficult. What does that even mean? Use lots of smileys? ;-)

Reply:
I'm fine with `should` and all. I think that "kind" is not about being flowery or full of smiles, but about taking the time to express feedback without empathy and compassion instead of judgement or dismissal.

Ultimately, these are intended to set expectations, and encourage us to strive to a higher standard than many projects, where maintainers are dismissive or even hostile to their users who are reporting bugs or asking for features.

Reply:
I like the edit, though personally I'd still go further by
- removing "all" - because smaller comments might not hit these points (e.g. "good job!" example), and just "feedback" makes it clearer we mean the overall set of comments
- changing "kind" to "not unkind", maybe - I think that's easier to be sure you're "not unkind", but it sounds more awkward so *shrugs*

Reply:
Other @zinc-collective/core folks want to weigh in? I'm hesitant to change it because I'd rather we have a high standard, but I am willing to tip towards softening if I get another voice advocating for it.

Reply:
I'm OK with the use of kind. Maybe we can include a footnote clarifying use of terms. Maybe phrasing along the lines of: "By kind, we don't mean foregoing discussion for the sake of being agreeable. We interpret kindness as the raising of feedback thoughtfully for clarity of ideas and behavior."

I'm someone who is prone to biting my tongue to not "raise issues" (and being seen as disruptive) so I do like the idea of saying it's OK to ask for discussion, as long as it's operating by community guidelines.